cmd2/cmd2.py

```diff
@@ -160,7 +160,7 @@ def parse_quoted_string(string: str, preserve_quotes: bool) -> List[str]:
         lexed_arglist = string
     else:
         # Use shlex to split the command line into a list of arguments based on shell rules
-        lexed_arglist = shlex.split(string, posix=False)
+        lexed_arglist = shlex.split(string, comments=False, posix=False)
```

Comment:
``False``  is the default value for ``comments``, so you don't need to pass it unless you think this makes it more self-documenting

Reply:
I did it for self documenting reasons